docs/user_manual/transfers.md

```diff
@@ -0,0 +1,101 @@
+includes any transfers for which you are a member of the recipient group and there are items that have not yet had
+their receipt or QC confirmed. The toolbar at the top of the table includes controls for working with transfers.
+
+
+
+## Creating Transfers
+
+### Sample and Library Receipt
+
+Receipt transfers are automatically created or updated when you enter a date of receipt and other receipt information
+when you are creating new samples and libraries. If the date of receipt, sender lab, and recipient group match an
+existing transfer that contains samples or libraries from the same project, the new items will be added to the existing
+transfer. If there is no existing transfer that matches, a new transfer is created.
+
+
+
+### Internal and Distribution Transfers
+
+Internal and distribution transfers can be created in several different ways.
+
+* On the Transfers list page, click the "Add" button in the toolbar
+* On the Samples, Libraries, Library Aliquots, or Pool list page, select the items to transfer and click the "Transfer"
+  button in the toolbar
+* On the Edit Sample, Edit Library, Edit Library Aliquot, or Edit Pool page, in the Transfers list, click the "Add"
+  button in the toolbar
+
+Any of these will take you to the Create Transfer page. In the top section of the page, fill in the transfer details.
+You can enter a recipient (free text) for distribution transfers, or select a recipient group for internal transfers.
+
+Below is a list of items to include in the transfer. If you used the "Add" button on the Transfers list page, the list
+will be empty; otherwise, the items you had selected will be included in the list already. You can add more items using
+the "Add" button. A dialog will ask which type of items you would like to add, and then allow you to enter a list of
+names, aliases, or barcodes for the items to add. Remove items from the transfer by selecting them in the list, then
+clicking the "Remove" button.
+
+When you are finished, click the "Save" button at the top right. Note that if you are creating a distribution transfer,
+each item will be marked as received automatically upon saving the transfer.
+
+Members of the sender group will be able to modify the transfer information and the included items using the Edit
+Transfer page.
+
+
+
+## Receiving Transfers
+
+For receipt and internal transfers, a member of the recipient group must confirm receipt and QC of each item. This can
+be done for receipt transfers during sample or library creation by setting the appropriate values for receipt
+confirmed, receipt QC, and receipt QC note. Internal transfers and receipt transfers for which these are not set will
+be in a pending state upon creation.
+
+If you are a member of a recipient group that has pending transfers, the "Transfers" link in the menu will be shown in
+bold text. The Pending tab of the Transfers list page includes any pending transfers for which you are a member of the
```

Comment:
The bold menu item as an alert was not intuitive to me without having read this manual

Reply:
Added a count of pending transfers in parentheses:

![image](https://user-images.githubusercontent.com/12240540/70917888-22836200-1fec-11ea-97e5-c933d9baac76.png)

I think this is much more intuitive now?

Reply:
:+1: